src/core.ts

```diff
@@ -258,13 +258,13 @@ const predefinedChildrenByUid = Object.fromEntries(
 export const HIDE_REGEX = /<%HIDE%>/i;
 
 export const getCustomWorkflows = () =>
-  window.roamAlphaAPI
+  window.roamAlphaAPI.data.fast
     .q(
-      `[:find ?s ?u :where [?r :block/uid ?u] [?r :block/string ?s] [?r :block/refs ?p] (or [?p :node/title "SmartBlock"] [?p :node/title "42SmartBlock"])]`
+      `[:find (pull ?r [:block/uid :block/string]) :where [?sb :node/title "SmartBlock"] [?old :node/title "42SmartBlock"] (or [?r :block/refs ?old] [?r :block/refs ?sb])]`
```

Comment:
There are some low hanging fruit plaguing sb/qb users that we can start resolving. This involves two:
- always use `data.fast.q` over `q`.
- reorder clauses to prioritize ones with high specificity first